cypress/integration/e2e/header-control-menu/return-to-now-button.spec.ts

```diff
@@ -0,0 +1,19 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+describe("'Come back to now' button", () => {
+  context("when schedule is loaded", () => {
+    before(() => {
+      cy.loadScheduleToMonth();
+      cy.get("[data-cy=switch-next-month]").click();
```

Comment:
Ten `cy.get` powinien być nie tutaj a w samym teście, ponieważ dotyczy konkretnie jego.

W przypadku jeśli dojdą nowe testy do tego spec'a, to takie kliknięcie nie będzie miało sensu